assets/enx-redirect/report/index.html

```diff
@@ -59,7 +59,7 @@ <h2 class="w-100 text-center">{{t $.locale "user-report.request-code-from"}}</h2
             <p>{{tDefault $.realmLocale (t $.locale "user-report.phone-number-instructions") "webReportPhoneNumberMessage"}}</p>
             <p>{{t $.locale "user-report.phone-storage"}}
             {{if $currentRealm.UserReportLearnMoreURL}}
-              <a href="{{$currentRealm.UserReportLearnMoreURL}}" target="_blank">{{t $.locale "user-report.learn-more"}}</a>
+              <a href="{{$currentRealm.UserReportLearnMoreURL}}" rel="noopener noreferrer" target="_blank">{{t $.locale "user-report.learn-more"}}</a>
```

Comment:
This is the important one, since this could be a PHA-controlled website, and we don't want to give them the ability to execute arbitrary javascript on our site.